tests/test_dask_image/test_ndmeasure/test_find_objects.py

```diff
@@ -0,0 +1,36 @@
+import dask.array as da
+import numpy as np
+import pandas as pd
+import pytest
+
+import dask_image.ndmeasure
+
+
+@pytest.fixture
+def label_image():
+    """Return small label image for tests.
+
+    dask.array<array, shape=(5, 10), dtype=int64, chunksize=(5, 5), chunktype=numpy.ndarray>
+
+    array([[  0,   0,   0,   0,   0,   0,   0, 333, 333, 333],
+            [111, 111,   0,   0,   0,   0,   0, 333, 333, 333],
+            [111, 111,   0,   0,   0,   0,   0,   0,   0,   0],
+            [  0,   0,   0, 222, 222, 222, 222, 222, 222,   0],
+            [  0,   0,   0,   0,   0,   0,   0,   0,   0,   0]])
+
+    """
+    label_image = np.zeros((5, 10)).astype(int)
+    label_image[1:3,0:2] = 111
+    label_image[3,3:-2] = 222
+    label_image[0:2,-3:] = 333
+    label_image = da.from_array(label_image, chunks=(5, 5))
+    return label_image
+
+
+def test_bounding_boxes(label_image):
+    result = dask_image.ndmeasure.find_objects(label_image).compute().compute()
```

Comment:
This is very irritating: I don't know why I need to call compute twice here, or how to fix that.

Reply:
Here's a copy of what I wrote in the dask slack about this problem:

> After running the fold operation, I get a dask.bag.core.Item as the result. If you call compute, you get a dask dataframe, but then you need to call compute again to, you know, compute the actual result. How can I avoid users having to call compute twice?Things I tried:
> * Putting one of the compute statements inside my own function. This isn't great, because it runs a bunch of the computation immediately.
> * Turning the dask.bag.core.Item into a dask dataframe. 
>    * There is a to_dataframe method for regular bags, but this is not available for dask bag Items.
>    * Dask bag Items do have a to_delayed method, so I could do that & then use the dask dataframe dd.from_delayed function. This would work great, except that Dask is doing some type checking and raising an error "Expected partition of type `pandas.core.frame.DataFrame` but got `dask.dataframe.core.DataFrame`". I've been trying to solve that by using the parent_meta kwarg in the dask.dataframe.utils.make_meta function - but it can't dispatch to a dask dataframe backend (only pandas and sparse backends are supported).
> 
> My current plan is to try creating a metadata dataframe without using the convenience make_meta function, and see how that goes.

> Update: nope, that didn't seem to help. I tried wrapping the pandas metadata object inside a dask dataframe, but the type mismatch error is still raised
> `meta = dd.from_pandas(dd.utils.make_meta([(i, object) for i in range(label_image.ndim)]), npartitions=1)`

> Oh, maybe I have it!
> In `dd.from_delayed` there is a `verify_meta` keyword argument, and I can set that to False to skip the partition type checking. 

> For reasons I don't understand, it still seems like you need to call compute twice on the output. Argh
> It's late, I'm going to leave this until Monday. For anyone who'd like more context, the PR I've been working on is here: https://github.com/dask/dask-image/pull/240

Reply:
I think problems can occur when mixing the delayed framework with dask arrays.

Replacing

```python
bag = db.from_sequence(arrays)
```
by
```python
    from dask import delayed, compute
    bag = db.from_delayed([delayed(compute)(ar) for ar in arrays])
```
makes it compute after a single call 😅. However this apart from being hacky might come with suboptimal graph optimization?

Reply:
For a basic example it seems nicely parallelized. I think we'd need it to be *very* suboptimal before I'd be happy about telling users to call compute twice, it's not a clean experience for them.

![image](https://user-images.githubusercontent.com/30920819/127441460-76163cd8-725e-4799-a7f4-753ebd8b14c3.png)

Reply:
Thanks a bunch for suggesting this @m-albert 

I was unhappy about this issue, and I think you've solved it nicely

Reply:
(Turns out we want to put this delayed compute call a bit later on, just before we tell it the result should be a dask dataframe. Works great!)